read-fonts/src/tables/glyf.rs

```diff
@@ -945,6 +958,52 @@ mod tests {
         );
     }
 
+    #[test]
+    fn simple_glyph_overlapping_contour_flag() {
+        let font = FontRef::new(font_test_data::VAZIRMATN_VAR).unwrap();
+        let loca = font.loca(None).unwrap();
+        let glyf = font.glyf().unwrap();
+        let glyph_count = font.maxp().unwrap().num_glyphs();
+        for gid in 0..glyph_count {
+            let glyph = match loca.get_glyf(GlyphId::new(gid), &glyf) {
+                Ok(Some(Glyph::Simple(glyph))) => glyph,
+                _ => continue,
+            };
+            if gid == 3 {
+                // Only GID 3 has the overlap bit set
+                assert!(glyph.has_overlapping_contours())
+            } else {
+                assert!(!glyph.has_overlapping_contours())
+            }
```

Comment:
This seems like the verbose version of  `assert_eq!(gid == 3, glyph.has_overlapping_contours())` ?

Reply:
Hm. Actually even then it's a bit more vulnerable to broken test data than I like; I propose we directly assert that only gid 3 has the bit:

`assert_eq!(vec![3], (0..glyph_count).filter(...retain only if has overlaps).collect())`

Reply:
I like this style as well. Fixed!